recipe/meta.yaml

```diff
@@ -0,0 +1,95 @@
+{% set version = "2.12.0" %}
+{% set name = "tensorflow-text" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/tensorflow/text/archive/v{{ version }}.tar.gz
+  sha256: 0de6d94d0b76a6050ae8c0ff618bbc5d9a646ef9929161149aad63974f8a56f2
+  patches:
+    - 0001-adjust-tf-version-range.patch
+    - 0002-make-other-minor-adjustments.patch
+    - 0003-do-not-check-for-bazel-version.patch
+    - 0004-skip-pull-tf-dependencies.patch
+    - 0005-fix-broken-text-filters.patch
+    - 0006-add-bazel-flags-for-macos-build.patch  # [osx]
+    - 0006-clean-up-after-bazel-temp-files.patch  # [linux]
+
+build:
+  number: 0
+  # skipping s390x and win-64 since we don't have tf for it
+  skip: True  # [s390x or win]
+  # skipping osx builds because bazel configuration for missing sources (bpe_model.cc, etc.)
+  # are too difficult to configure in bazel
+  skip: True  # [osx]
+  skip: True  # [py>=312]
+  missing_dso_whitelist:                   # [linux]
+    - $RPATH/libtensorflow_framework.so.2  # [linux]
+    - '*/libdl.so.*'                       # [linux]
+    - '*/libpthread.so.*'                  # [linux]
+    - '*/libm.so.*'                        # [linux]
+    - '*/libstdc++.so.*'                   # [linux]
+    - '*/libgcc_s.so.*'                    # [linux]
+    - '*/libc.so.*'                        # [linux]
+    - '*/ld-linux-aarch64.so.*'            # [linux]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - bazel
+    - git  # [not win]
+    - patch  # [not win]
+  host:
+    - python
+    - pip
+    - setuptools
+    - wheel
+    # The version of tensorflow has to match the version of tensorflow-text
+    - tensorflow {{ version }}
+    #- _tflow_select 2.3.0 mkl
+  run:
+    - python
+    - {{ pin_compatible("tensorflow", min_pin="x.x.x", max_pin="x.x") }}
+    - tensorflow-hub >=0.8.0
+    #- _tflow_select * mkl
+
+test:
+  source_files:
+    - oss_scripts
+    - WORKSPACE
+    - tensorflow_text/python/ops
+    - tensorflow_text/python/ragged
+    - tensorflow_text/python/numpy
+    - tensorflow_text/python/metrics
+    - tensorflow_text/python/benchmarks
+    - third_party
```

Comment:
yes we do, every single one of the _if_ we want to run _all_ tests. Currently the tests give me quite some headaches with `bazel`, that's why I'm trying to at least find those out of _all_ that don't. In that case I would not need all these files, but only some.
But also, I'm not so sure if I would like to see those tests in the `build.sh`, since `build.sh` is supposed to be for the building phase.

Reply:
>But also, I'm not so sure if I would like to see those tests in the build.sh, since build.sh is supposed to be for the building phase.

We do this pretty often with compiled projects.

Reply:
right now I may be able to run most tests without additional test dependencies, but if some dependencies are specific to tests, I will need to add them here and run `run_tests.sh` from here. Otherwise I will move it to the build stage.